### ISSUE TITLE
Fedora 36 KDE Support Lacking

### ISSUE THREAD

**Issue by rbreaves** [rbreaves/kinto#674]
**Describe the bug**
Dependency missing
vte291-devel.

keyboard input stops working due to signal error with xkeysnail.

**Comment by RedBearAK** [comment]
@rbreaves 

Dude, what the… 

I’m literally just in the middle of setting up F36 KDE spin on a 2007 MacBook, in anticipation of testing a Kwin script that simulates macOS application switching in Plasma. 

Anyway, ran into the same issue and solved it with the “vte291” package. No “devel” needed. 

The installer succeeds without it, but should probably install “python3-wheel” also. 

Are you testing the environment variable “$XDG_SESSION_TYPE” for “x11”? I didn’t run into that problem since I remembered to change the session to X11 right after install.

**Comment by rbreaves** [comment]
Yea devel probably not needed & the latter well I was in wayland by accident - shouldn’t happen like that though.

**Comment by Tokarak** [comment]
I'm running into this issue on Fedora 43, KDE. One issue was first resolved by installing "vte291". After that, I ran into xkeysnail failing; this is presumably due to Wayland. I am not bothering to change my session to X11.

**Comment by RedBearAK** [comment]
@Tokarak 

My project (Toshy) was initially based on Kinto but uses a fork-of-a-fork of xkeysnail with lots of fixes and new features added, and has full support for all the Wayland environments that I am aware of, a long process which started with the GNOME and KDE 5/6 Wayland sessions. 

I'm using Plasma 6.6 on Fedora 43 and Tumbleweed myself, with no issue. Several dozen different Linux distros are currently supported by the Toshy setup script. You may wish to check it out. 

https://github.com/RedBearAK/Toshy (shorter URL https://toshy.app)

**Comment by Tokarak** [comment]
> @Tokarak 
> 
> My project (Toshy) was initially based on Kinto but uses a fork-of-a-fork of xkeysnail with lots of fixes and new features added, and has full support for all the Wayland environments that I am aware of, a long process which started with the GNOME and KDE 5/6 Wayland sessions. 
> 
> I'm using Plasma 6.6 on Fedora 43 and Tumbleweed myself, with no issue. Several dozen different Linux distros are currently supported by the Toshy setup script. You may wish to check it out. 
> 
> https://github.com/RedBearAK/Toshy (shorter URL https://toshy.app)
> 

Yup, thanks, I already installed yours from another one of your ads. Works just fine, after a short read!
